configuration/outbound.md

```diff
@@ -0,0 +1,160 @@
+# Outbound push [beta]
+
+*Outbound is only available in 3.5.0 and above*
+
+Outbound push allows configurers to have single reports generated by CHWs or others translate into triggering an external web requests using the data in that report. For example, upon receiving a referral report you could send that referral to an external facility system that will manage and process that event.
+
+This feature is experimental, and may be replaced by some third party technology over time as we discover the feature-set we're looking for.
+
+## Configuration
+
+For outbound pushes to occur, you must enable the `mark_for_outbound` transition in config:
```

Comment:
Should we explicitly mention this is in the settings doc, or set in a medic-conf project's `app_settings.json`?

Reply:
I'll link it back to the transition config, whose job it is to explain how to configure transitions